spec/steps/show_me_the_mails_spec.rb

```diff
@@ -93,4 +93,26 @@
       expect { step.run }.to output(expected_output).to_stdout
     end
   end
+
+  context "with attachments" do
+    it 'includes the filenames of the attachments' do
+      mail = Mail.new do
+        attachments['attached_file.pdf'] = File.open("tests/shared/public/fixture_files/attachment.pdf", "w") {}
```

Comment:
Reicht nicht einfach `File.open("tests/shared/public/fixture_files/attachment.pdf")`? 
Wir müssen ja nichts in die Datei schreiben.